src/components/ViewBook.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const ViewBook = ({ book }) => (
+  <div className="booklet">
+    <p className="title">
+      Title:
+      {book.title}
+    </p>
+    <p className="author">
+      Author:
+      {book.author}
+    </p>
+  </div>
+);
```

Comment:
- [OPTIONAL] Please add a button for removing books. 
       - The button wouldn't remove the books yet, this is just a placeholder for now.

Reply:
Thank so much, @mmsesay